CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/AnalysisViewController.swift

```diff
@@ -133,30 +142,14 @@ import UIKit
     
     public override func loadView() {
```

Comment:
we need to remove `loadView()` explanation is here https://developer.apple.com/documentation/uikit/uiviewcontroller/1621454-loadview